tests/docs/running-perf-tests.md

```diff
@@ -144,3 +144,150 @@ make test-perf-all
 To keep the build infrastructure simple, Dapr uses dapr-test GitHub Actions Workflow to run e2e tests using one of AKS clusters. A separate workflow also runs E2E in KinD clusters.
 
 Once a contributor creates a pull request, E2E tests on KinD clusters are automatically executed for faster feedback. In order to run the E2E tests on AKS, ask a maintainer or approver to add /ok-to-perf comment to the Pull Request.
+
+
+
+
+# Setup Required For Visualising Performance Test Metrics
+
+The setup in AKS requires three servers for:
+ - Prometheus 
+ - Pushgateway
+ - Grafana
+ 
+All the servers should be installed in the same namespace to ensure effective communication between them.
+ 
+ * Create a namesapce
+  
+    ```bash
+    kubectl create namespace <namespace-name>
+    ```
+ * Set this as the current namespace
+    
+    ```bash
+    kubectl config set-context --current --namespace=<namespace-name>
+    ```
+ * Check if the namespace is set
+  
+    ```bash
+    kubectl config view | grep namespace:
+    ```
+
+## Setup for Prometheus Server
+
+* Prometheus can be installed on AKS using the following commands:
+ 
+  ```bash
+  helm repo add prometheus-community https://prometheus-community.github.io/helm-charts 
+  helm repo update
+  helm install prometheus prometheus-community/prometheus
+  ```
+* While installing this, there may be an error saying that some clusterrole or clusterrolebindings already exist. In that case delete the clusterrole or clusterrolebinding
+
+  ```bash
+  kubectl delete clusterrole <clusterrole-name>
+  kubectl delete clusterrolebinding <clusterrolebinding-name>
+  ```
```

Comment:
We can leave out this details, and simply link the official Prometheus docs.

Reply:
Applied.